src/components/SpecialResolution/CreateSpecialResolution.vue

```diff
@@ -300,23 +297,27 @@ export default class CreateSpecialResolution extends Mixins(DateMixin) {
    * True if date is >= the minimum (ie, today) and <= the maximum (ie, the 10th day).
    * This is used for Vue form validation (in Date Rules above).
    */
-  isBetweenDates (minDate: string, maxDate: string, dateStrToValidate: string): boolean {
+  private isValidDateRange (minDate: Date, maxDate: Date, dateStrToValidate: string): boolean {
     if (!dateStrToValidate) { return true }
-    return (new Date(dateStrToValidate) >= new Date(minDate)) && (new Date(dateStrToValidate) <= new Date(maxDate))
+    // only compare year/month/day (ignore time)
+    const date = new Date(dateStrToValidate) // Input is in the format of MM dd, yyyy
+    return (date >= minDate && date <= maxDate)
```

Comment:
"minDate" and "maxDate" are correctly in UTC (according to the Date Mixin method used to create them) but I think "date" is set according to the browser's local time offset.

Easy solution: use `yyyyMmDdToDate(dateStrToValidate)`.

Reply:
For signatory:

minDate is in PST
date is in PST
maxDate arrive in PST
![image](https://user-images.githubusercontent.com/3484109/187521466-56a53c40-e5f6-49b7-a2d0-2fb898ef1c12.png)


For Resolution Date:

minDate is UTC ( doesn't matter it's getting changed )
date is in PST
maxDate is in PST
![image](https://user-images.githubusercontent.com/3484109/187521572-7adcbd71-b165-4353-ba07-13d3c547fa9b.png)

Reply:
`   const date = this.yyyyMmDdToDate(dateStrToValidate) // Input is in the format of MM dd, yyyy`

results in 

`date: null`

Reply:
Changing timeclock on my computer, and retesting date

Reply:
Here are the different types of dates in our UIs:
1. "yyyy-mm-dd" are assumed to be Pacific date
2. "yyyy-mm-ddThh:mm:ss.xxxZ" are assumed to be UTC (ISO format)
3. "yyyy-mm-ddThh:mm:ss.xxx+00.00" are assumed to be UTC (API format)
4. Date objects, you have to careful with
   - if a Date Mixin method was used to create it, it should be in UTC internally (and display according to the browser's timezone)
   - if `Date()` was used to create it, it reverse-offsets according to the browser's timezone and stores internally as "UTC", but you can see here the problem that might occur if the browser's time offset is incorrect/spoofed

Reply:
> ` const date = this.yyyyMmDdToDate(dateStrToValidate) // Input is in the format of MM dd, yyyy`
> 
> results in
> 
> `date: null`

Ah, `yyyMmDdToDate` doesn't support your internal format.

Reply:
PS When you say a Date object is "in PST", I believe it's actually stored as UTC and displayed in your browser's timezone.

Reply:
> Changing timeclock on my computer, and retesting date

Try a different timezone, too.

Reply:
![image](https://user-images.githubusercontent.com/3484109/187525477-a943448b-ddb2-46d9-a525-64175f20646b.png)

Reply:
^^ using `const date = new Date(dateStrToValidate)` and your workstation/browser set to different timezones?

Explanation: In your example, JS magically knows the correct UTC offset from the string "Aug 30, 2022"... I assume it uses your workstation/browser offset for this, which might differ. However, we want a fixed offset -- always Pacific -- for displayed dates.

Reply:
dateStrToValidate is coming from `DatePickerShared`

Reply:
Closing this thread. This is continued in another conversation below.